pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -446,6 +451,12 @@ func (configgen *ConfigGeneratorImpl) createGatewayHTTPFilterChainOpts(node *mod
 				},
 				ServerName:          EnvoyServerName,
 				HttpProtocolOptions: httpProtoOpts,
+				// Per https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto#route-virtualhost
+				// we can only have one wildcard. Ideally, we want to match any port, as the host
+				// header may have a different port (behind a LB, nodeport, etc). However, if we
+				// have a wildcard domain we cannot do that since we would need two wildcards.
+				// Therefore, we always strip host port.
```

Comment:
This should be only added for proxies GE 1.10?

Reply:
isn't this supported in 1.9?

Reply:
It is there 1.17 release of Envoy. So I think we should be good